README.md

```diff
@@ -71,3 +71,13 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets
+
+There is a nice project at
+[bittner/lego-mindstorms-ev3-comparison](https://github.com/bittner/lego-mindstorms-ev3-comparison)
+highlighting differences between LEGO MINDSTORMS EV3 Home Edition (#31313) and LEGO
```

Comment:
I think we should put code formatting around numbers so that GitHub doesn't make them links :laughing:

Reply:
It will be a while before github will be able to link to those issues, but I did it anyway :)